internal/cluster/transition.go

```diff
@@ -402,8 +402,13 @@ func (th *transitionHandler) PostRefreshCluster(reason string) stateswitch.PostT
 			err            error
 			updatedCluster *common.Cluster
 		)
-		updatedCluster, err = updateClusterStatus(logutil.FromContext(params.ctx, th.log), params.db, *sCluster.cluster.ID, sCluster.srcState, *sCluster.cluster.Status,
-			reason)
+
+		//if no changes need to be made return
```

Comment:
please change the remark to: "There is no need to update DB if nothing was changed". The current remark is not correct